packages/lib/src/components/Card/components/CardInput/types.ts

```diff
@@ -122,7 +121,7 @@ export interface CardInputProps {
     showPayButton?: boolean;
     showStoreDetailsCheckbox?: boolean;
     showWarnings?: boolean;
-    specifications?: Specifications;
+    specifications?: AddressSpecifications;
```

Comment:
fixed the type here, the value of `specifications` coming from props is not the class rather its the `AddressSpecifications` object